aws/monolith-terraform/xebialabs/xld-environment.yaml.tmpl

```diff
@@ -0,0 +1,33 @@
+{{$app := .AppName | kebabcase}}
+apiVersion: xl-deploy/v1
+kind: Infrastructure
+spec:
+- name: {{$app}}
+  type: core.Directory
+  children:
+    - name: localhost
+      type: overthere.LocalHost
+      os: UNIX
+      children:
+        - name: terraform-client
+          type: terraform.TerraformClient
```

Comment:
lets add below so that user can update easily if needed
```
      path: '/usr/local/bin' # for local installation
      workingDirectory: '/tmp' # for local installation
```